### DISCUSSION TITLE
Vcpu limit exceeded

### DISCUSSION THREAD

**Discussion by LavanyaRamesh319** [learndevopsonline/learndevopsonline#1910] [Q&A]
I am getting error as Vcpu limit exceeded. I have requested for Quota increase also and it got approved. But still I am getting error. could you please help me to resolve this issue.

Workstation ip address -- **54.86.97.206**
Jenkins ip address -- **184.72.82.66**

![image](https://github.com/learndevopsonline/learndevopsonline/assets/103929459/db3fd56d-ffd7-41e3-a8da-8c21617ff26a)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/103929459/99009b9b-c26b-4b07-90ef-7fe97cf76cd7)

**Comment by LavanyaRamesh319** [comment]
Hi @chaithanya1812 ,

Now I am getting different output

![image](https://github.com/learndevopsonline/learndevopsonline/assets/103929459/3facad22-0e54-471c-9d86-08bfe5da18b8)

I am really stuck with the issue.

**Comment by chaithanya1812** [comment]
In lab session we discussed, It's aws side issue.  aws facing out of capacity in some zones currently which we are using now. wait for one day
once aws done their capacity planning increment, then we are able to work.
